src/src/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -756,7 +756,7 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
                     Logger.v(TAG, "Error info:" + errCode + " " + errMessage + " for requestId: "
                             + requestId + correlationInfo);
                     waitingRequestOnError(waitingRequest, requestId, new AuthenticationException(
-                            ADALError.SERVER_INVALID_REQUEST, errCode + " " + errMessage));
+                            ADALError.SERVER_INVALID_REQUEST, errCode + " " + errMessage + correlationInfo));
```

Comment:
Will there be a whitespace missing between errMessage + correlationInfo concatination?